### PR TITLE
various UI tweaks to clean up the way we do searchable address inputs on mobile

### PR DESCRIPTION
**Pull request by travisbloom-cb**
## What changed? Why?

There are a number of buggy issues with our address autocomplete on mobile. See comments for addressed bugs.

### Mobile Web

| Old 👴         | New 👶         |
| -------------- | -------------- |
| image | image |

## How has it been tested?

- [X] Locally
- [ ] Vercel Preview Branch
- [ ] Unit test
- [ ] Functional test

## Change management

type=routine 
risk=low 
impact=sev5

### CODE REVIEW

File: src/components/ui/combobox.tsx [hunk 1]
```diff
@@ -69,25 +69,23 @@ export function Combobox<T>({
...
-                options,
-                getOptionLabel,
-                getOptionKey,
-                ...inputProps,
-              }}
-            />
-          </div>
-        </DrawerContent>
-      </Drawer>
+      <Dialog analytics={wrappedAnalytics} onOpenChange={setOpen} open={open}>
```
Comment (travisbloom-cb) [inline src/components/ui/combobox.tsx:72]: The drawer component has a weird bug that I can't seem resolve where, on iOS devices, when a drawer is opened over a dialog (like in the email your rep flow) every other time you open the drawer, it is unclickable and will close when tapped. If we can figure out why and fix, we can revert this change

File: src/components/ui/command.tsx [hunk 2]
```diff
@@ -43,22 +43,26 @@ const CommandInput = React.forwardRef<
...
   }
->(({ className, onClear, commandValue, ...props }, ref) => (
-  <div className="flex items-center border-b px-3" cmdk-input-wrapper="">
-    <Search className="mr-2 h-4 w-4 shrink-0 opacity-50" />
+>(({ className, onClear, commandValue, containerClassName, ...props }, ref) => (
+  <div
+    className={cn('relative flex items-center border-b', containerClassName)}
+    cmdk-input-wrapper=""
+  >
+    <Search className="pointer-events-none absolute left-3 h-4 w-4 opacity-50" />
```
Comment (travisbloom-cb) [inline src/components/ui/command.tsx:53]: This ensures that when you click the magnifying glass icon, it focuses the input

File: src/components/ui/command.tsx [hunk 3]
```diff
@@ -43,22 +43,26 @@ const CommandInput = React.forwardRef<
...
-        'flex h-11 w-full rounded-md bg-transparent py-3 text-sm outline-none placeholder:text-muted-foreground disabled:cursor-not-allowed disabled:opacity-50',
+        'flex h-11 w-full rounded-md bg-transparent py-3 pl-10 pr-3 text-sm outline-none placeholder:text-muted-foreground disabled:cursor-not-allowed disabled:opacity-50',
         className,
       )}
       ref={ref}
       {...props}
     />
     {Boolean(props.value || commandValue) && (
-      <button className="py-3" onClick={onClear}>
+      <button className="py-3 pl-3 pr-[18px]" onClick={onClear}>
```
Comment (travisbloom-cb) [inline src/components/ui/command.tsx:63]: made clear click target bigger

File: src/components/ui/googlePlacesSelect/index.tsx [hunk 4]
```diff
@@ -57,8 +57,6 @@ export const GooglePlacesSelect = React.forwardRef<
           value={triggerProps.value?.description || inputProps.placeholder || 'select a location'}
           {...inputProps}
           readOnly
-          // There's a weird bug where, because the input is type="button", on mobile a long address string will overflow the entire page
-          type="input"
```
Comment (travisbloom-cb) [inline src/components/ui/googlePlacesSelect/index.tsx:61]: i can no longer reproduce this....if someone else can lmk

File: src/components/ui/command.tsx [hunk 5]
```diff
@@ -43,22 +43,26 @@ const CommandInput = React.forwardRef<
...
     commandValue?: unknown
+    containerClassName?: string
   }
->(({ className, onClear, commandValue, ...props }, ref) => (
-  <div className="flex items-center border-b px-3" cmdk-input-wrapper="">
-    <Search className="mr-2 h-4 w-4 shrink-0 opacity-50" />
+>(({ className, onClear, commandValue, containerClassName, ...props }, ref) => (
+  <div
+    className={cn('relative flex items-center border-b', containerClassName)}
+    cmdk-input-wrapper=""
```
Comment (paulovictor-cb) [inline src/components/ui/command.tsx:51]: This is supposed to be empty?

**Review by paulovictor-cb — APPROVED** [review]

File: src/components/ui/combobox.tsx (hunk 1 above)
Comment (lucasrmp-cb) [reply src/components/ui/combobox.tsx:72]: Will that "fullscreen dialog" task affect this in any way? I think it'd be a little weird if this became full screen

**Review by lucasrmp-cb — DISMISSED** [review]

**Pull request merged**